awx/main/migrations/0114_v370_remove_deprecated_manual_inventory_sources.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 2.2.11 on 2020-04-03 00:11
+
+from django.db import migrations, models
+
+
+def remove_manual_inventory_sources(apps, schema_editor):
+    '''Previously we would automatically create inventory sources after
+    Group creation and we would use the parent Group as our interface for the user.
+    During that process we would create InventorySource that had a source of "manual".
+    '''
+    InventoryUpdate = apps.get_model('main', 'InventoryUpdate')
+    InventoryUpdate.objects.filter(source='').delete()
```

Comment:
@wenottingham @AlanCoding is it reasonable to just delete these?

It looks like *today* inventory sources cascade their deletes to their associated updates.